Cargo.toml

```diff
@@ -90,7 +90,7 @@ openssl = "0.10"
 tokio-openssl = "0.2"
 hyper = { version = "0.12", default-features = false, features = ["runtime"] }
 keys = { path = "components/keys" }
-kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
+kvproto = { git = "https://github.com/innerr/kvproto.git", default-features = false, branch = "issue-6690" }
```

Comment:
You can replace it with [patch](https://doc.rust-lang.org/edition-guide/rust-2018/cargo-and-crates-io/replacing-dependencies-with-patch.html#replacing-dependencies-with-patch) in the root `Cargo.toml` only.

Reply:
Yes I did use patch locally at first. The purpose here is for pass CI.
We could check all things are OK, then merge related PR in kvproto, then reverse this dependency, then finally merge this PR